manifests/init.pp

```diff
@@ -172,6 +177,7 @@
   $puppetboard_proxy_url = hiera('puppetboard_proxy_url')
   file { '/etc/init.d/puppetboard' :
     content => template("puppetdb/etc/init.d/puppetboard.erb"),
+    mode    => 0755, 
   }
   
```

Comment:
The two changes above are already in puppet.  I guess I didn't update the subtree.